### PR TITLE
Add farm service

### PR DESCRIPTION
**Pull request by Biruk-Tassew**

### CODE REVIEW

**Review by hello-world-was-taken — APPROVED** [review]

**Review by brukted — APPROVED** [review]

File: app/src/main/java/com/enterprise/agino/data/remote/api/GeocodeService.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Url
+
+interface GeocodeService {
+
+    @GET
+    suspend fun latLngToPlace(@Url url: String): Response<ReverseGeocodeResponse>
+
+}
```
Comment (brukted) [inline app/src/main/java/com/enterprise/agino/data/remote/api/GeocodeService.kt:13]: Delete this file, I don't think we will need it.

File: app/src/main/java/com/enterprise/agino/data/remote/dto/ReverseGeocodeResponse.kt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+data class BoundingBox (
+    val northEast: String,
+    val southWest: String,
+    val entity: String
+)
+
+data class DataSources (
+    val geometry: Geometry
+)
```
Comment (brukted) [inline app/src/main/java/com/enterprise/agino/data/remote/dto/ReverseGeocodeResponse.kt:35]: This one too

**Review by brukted — CHANGES_REQUESTED** [review]

**Review by brukted — APPROVED** [review]

**Comment by Biruk-Tassew** [comment]
[Integrate add farm](https://trello.com/c/rPDcpj5Z/60-integrate-add-farm)

**Comment by nawrazi** [comment]
[Integrate add farm feature](https://trello.com/c/VaQzH7iq/47-integrate-add-farm-feature)

**Pull request merged**